### DISCUSSION TITLE
🌐 New icon

### DISCUSSION THREAD

**Discussion by nitin2953** [U-C-S/Hurl#157] [General]
![web-α](https://user-images.githubusercontent.com/34852979/161196701-3a81dc1b-4f97-4a30-915e-7160d9d93563.png)
credit: https://uxdesign.cc/interaction-design-is-more-than-just-user-flows-and-clicks-4cc37011418c#24ab

**Comment by U-C-S** [comment]
Intresting Article and Icon is really good....

Might actually use it. Previously, I was planning on making one myself with Figma.

**Comment by U-C-S** [comment]
Will come back to this once I have `v0.9` ready...

**Comment by Jay-o-Way** [comment]
I wouldn't "steal" a random icon from somewhere.
I have an idea in mind. Use the basic and familiar "wireframe", and then give it a circular rainbow colored fill. The different colors would be a metaphor for the different browsers.

**Comment by U-C-S** [comment]
Interesting.... I got a few ideas following yours. Maybe an another "experiment" for the coming weekends, I will try sketch-up a few with my rusty photo editing skill.

**Comment by Jay-o-Way** [comment]
Quick concept:
![Concept 1](https://github.com/U-C-S/Hurl/assets/65828559/b938dc43-4ec0-4feb-bb64-439516686130)

**Comment by stamminator** [comment]
Personally, I like the current icon. Definitely keep the cursor in the icon, as it makes the intention clear. Maybe it could be cleaned up a bit to not be so pixelated, but other than that, it's good.

Subjectively, I find the heavy outlines, solid colors, and mild non-gradient shadows used in apps like WSL Toolbox to be very appealing...

![icon](https://github.com/user-attachments/assets/86e27663-80b9-459f-b449-ebcfcfa63e01)

... but if you dislike that, you can't go wrong with soft, matte, gradient look of Fluent-style icons. Here are some good examples I picked from Windows 11's imageres.dll and the Segoe UI Emoji font:

![icon](https://github.com/user-attachments/assets/947120e5-3041-46f3-a557-46467e19151e)
![Level slider](https://github.com/user-attachments/assets/63b66b55-c3d2-4d3f-9ddd-ee01287f23fe)
![Printer](https://github.com/user-attachments/assets/0586b2ff-1757-4b3b-82ec-f62c7ee806e0)

I've used Fiverr to commission icon designs for pretty cheap in the past. Might be worth a look.

**Comment by U-C-S** [comment]
Me and one of my friends were trying to designing one ourselves (or probably like he took the commission). already have a few prototypes, but too shy to put them out haha.

We are trying for something abstract, fluent-styled, matte color like the typical win11 inbox apps.
